portal/exercise_diet/templates/exercise_diet/exercise-diet_portal.html

```diff
@@ -46,5 +43,12 @@ <h6><span>{{_("EXERCISE / DIET / RECIPES")}}</span></h6>
 	</div><!-- END #Exercise-Diet-Portal -->
 </main>
 {% endblock %}
+{% block document_ready %}
+  <script>
```

Comment:
@achen2401 
Would you be able to move this into `exercise_diet.js`?

Reply:
@ivan-c  ivan, I have removed the inline script here to exercise_diet.js.  thank u for the reminder.